src/__tests__/transformers/Streamable.js

```diff
@@ -0,0 +1,72 @@
+import cases from 'jest-in-case';
+
+import plugin from '../../';
+import { getHTML, shouldTransform } from '../../transformers/Streamable';
+import {
+  cache,
+  getMarkdownASTForFile,
+  mockFetch,
+  parseASTToMarkdown,
+} from '../helpers';
+
+jest.mock('node-fetch', () => jest.fn());
+
+const expectedHtmlResult = `<iframe class="streamable-embed" src="https://streamable.com/o/bx960" frameborder="0" scrolling="no" width="1920" height="1080" allowfullscreen></iframe>`;
+
+cases(
+  'url validation',
+  ({ url, valid }) => {
+    expect(shouldTransform(url)).toBe(valid);
+  },
+  {
+    'non-streamable url': {
+      url: 'https://no-streamable-url-here.com',
+      valid: false,
+    },
+    'streamable-url-with-valid-embed-ending': {
+      url: 'https://streamable.com/bx960',
+      valid: true,
+    },
+    'streamable-url-with-s-valid-embed-ending': {
+      url: 'https://streamable.com/s/bx960',
+      valid: true,
+    },
+    'streamable-url-with-o-valid-embed-ending': {
+      url: 'https://streamable.com/s/bx960',
+      valid: true,
+    },
+    'streamable-url-with-long-embed-ending': {
+      url: 'https://streamable.com/s/bx960/hcobuo',
+      valid: true,
+    },
+  }
+);
+
+test('Gets the correct Streamable iframe', async () => {
+  mockFetch(expectedHtmlResult);
```

Comment:
Inline `expectedHtmlResult` with a separate `class` for the `iframe` & `plugin` test, to be sure we're mocking everything correctly (see Twitter test).